src/Components/OATPropertyEditor/PropertySelector.tsx

```diff
@@ -191,17 +191,17 @@ const PropertySelector = ({
                 <Separator styles={propertySelectorSeparatorStyles} vertical />
                 <div className={propertyInspectorStyles.propertyTagsWrapFirst}>
                     {propertySelectorData.propertyTags.primitive.map(
-                        (tag, i) => (
+                        (tag, index) => (
```

Comment:
Index isn't a great key. Ideally we use an actual identifying property on the tag.